src/docs/rules/should-specify-forbid-unknown-values.md

```diff
@@ -1,12 +1,11 @@
-### Rule: validation-pipe-should-forbid-unknown
+### Rule: should-specify-forbid-unknown-values
 
 This checks when if you are setting ValidationPipe parameters you set forbidUnknownValues to true.
 
 The rule is ignored if you use a spread operator to pass options to the validation pipe constructor.
 
-There is a CVE for class-transformer when using random javascript objects. You need to be careful about configuring the ValidationPipe in NestJs. See
-https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-18413
-https://github.com/typestack/class-validator/issues/438
+There is a CVE for class-transformer when using random javascript objects. You need to be careful about configuring the ValidationPipe in NestJs.
+See <https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-18413>, <https://github.com/typestack/class-validator/issues/438> & <https://github.com/nestjs/nest/issues/10683>.
```

Comment:
added link to the nestjs issue which explains that NestJs sets the default to `false` again (overwriting the class-validator default).